src/Servers/HttpSys/test/FunctionalTests/Listener/ServerTests.cs

```diff
@@ -123,6 +125,20 @@ public async Task Server_SetRejectionVerbosityLevel_Success()
             }
         }
 
+        [ConditionalFact]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/pull/20718#issuecomment-618758634")]
```

Comment:
You can target specific queues, if this only is an issue on Windows queues, it might be best to use the OS skip on windows

Reply:
All HttpSys tests only run on Windows.